### PR TITLE
function name on index

### PR DESCRIPTION
**Pull request by pedrofurst**
* Fixed function name on index.

### CODE REVIEW

**Comment by cacontinuos** [comment]
By analyzing the blame information on this pull request, we identified @fernahh and @caarlos0 to be potential reviewers

File: index.html [hunk 1]
```diff
@@ -92,7 +92,7 @@ <h2 class="semi-bold-title">Usage</h2>
               </tr>
               <tr>
                 <td class="col1"></td>
-                <td class="col34">CreditCard#isValidateSecuryCode()</td>
+                <td class="col34">CreditCard#isValidSecuryCode()</td>
```
Comment (VictorBogo) [inline index.html]: não seria `isValidSecurityCode`?

**Comment by cacontinuos** [comment]
-1

PR should have a **good** description!

File: index.html (hunk 1 above)
Comment (pedrofurst) [inline index.html]: https://github.com/ContaAzul/creditcard.js/blob/48e6fdd4d3fb55d4b9a2b0a4760ec24361567073/src/creditCard.js#L36

Comment (pedrofurst) [inline index.html]: tinha percebido, mas tava em dúvida.. vou arrumar tbm

**Pull request closed without merging**